File: Faktöriyel.java
import java.util.Scanner;

public class FaktoriyelHesaplama {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int sayi;
        int faktoriyel1 = 1;
        
        System.out.println("Bir sayı giriniz: ");
        
        sayi = scanner.nextInt();
        
        for (int i = 1; i <= sayi; i++) {
            faktoriyel1 = faktoriyel1 * i;
        }
        
        System.out.println("Faktoriyeli: " + faktoriyel1);
        
        scanner.close();
    }
}

File: Faktöriyel.cpp
#include <iostream>
using namespace std;

int main()
{
	int sayi;
	int faktoriyel1 = 1;

	cout << "Bir sayı giriniz: " << endl;
	cin >> sayi;

	for (int i = 1; i <= sayi; i++)
	{
		faktoriyel1 = faktoriyel1 * i;
	}

	cout << "Faktoriyeli: " << faktoriyel1 << endl;
}
